tests/app/questionnaire/routing/test_operators.py

```diff
@@ -229,6 +229,7 @@ def test_operation_in(operands, expected_result):
         [([0.5, 1], [0.3, 1]), False],
         [([1, 1.5, 3, 5], [1.5, 3, 5, 7]), False],
         [([None, "No"], ["Yes", "No"]), False],
+        [(["Yes", "No"], ("Yes", "No")), True],
```

Comment:
Minor, but these should be moved up to the group of tests for True.

Reply:
Ok 👍